partner_tlr_update/models/res_partner.py

```diff
@@ -89,19 +89,32 @@ def update_tlr_data(self):
                 response = api_client.get_tjansteleverantor(self.legacy_no)
                 if response.status_code == 200:
                     self.parse_xml_tjansteleverantor_data(response.text)
-                response = api_client.get_utforande_verksamhet_id(
-                    self.legacy_no)
+                    _logger.info("PARSED XML")
```

Comment:
Do we need all the _loggers for debbuging in pre-production enviorment?

Reply:
Might be helpful if something goes wrong even if it's maybe a bit excessive currently, they should be removed once it's tested and working.
It's not like it'll spam the log constantly, and even when it runs the logging isn't spitting out anything too big.